docs/content/Schema/pre-aggregations.md

```diff
@@ -643,14 +650,29 @@ new tables in the source database. For external pre-aggregations, these source
 tables are temporary - once downloaded and uploaded to the external database,
 they are cleaned up.
 
+### Known limitations of internal pre-aggregations
+
+Internal pre-aggregations are not considered production-ready due to several
+shortcomings, as noted below.
+
+**Concurrency:** Databases (especially RDBMs) generally cannot handle high
+concurrency without special configuration.
+
+**Latency:** Data warehousing solutions (such as BigQuery or Redshift) are often
+slow to return results.
+
+**Cost:** Some databases charge by the amount of data scanned for each query
+(such as AWS Athena and BigQuery). Repeatedly querying for this data can easily
+rack up costs.
+
```

Comment:
You've coined it, @hassankhan 👍

 I wonder if we can/should add some quantitative data here so it's clear what level of concurrency is considered high or what latency is practically unachievable with query engines...

Reply:
Absolutely, and I was hoping you might be able to help me with that 😅 Maybe we can discuss on Slack?

Reply:
@hassankhan @igorlukanin I like the idea! Have you discussed it in Slack?